frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/storage-and-nodes-step.tsx

```diff
@@ -109,9 +107,9 @@ export const StorageAndNodes: React.FC<StorageAndNodesProps> = ({ state, dispatc
   }, [cpu, dispatch, memory, nodesCount]);
 
   React.useEffect(() => {
-    const isFlexibleScaling: boolean = scNodeNamesLength && zonesCount < 3;
+    const isFlexibleScaling: boolean = !!(nodesCount && zonesCount < 3);
```

Comment:
shouldn't this be !!(nodesCount>=3 && zonesCount < 3) otherwise even one node is selected flexible scaling will get enabled.